tools/package.json

```diff
@@ -5,7 +5,8 @@
 	"scripts": {
 		"check:type": "tsc",
 		"test:ci": "vitest run",
-		"check:lint": "eslint ."
+		"check:lint": "eslint .",
+		"test:file": "node -r esbuild-register test/run-test-file.ts"
```

Comment:
I was literally thinking of asking if you could make it TS and add it to `tools` package!!